tasks/ca/statcan/geo.py

```diff
@@ -160,9 +177,11 @@ def columns(self):
     def populate(self):
         session = current_session()
         session.execute('INSERT INTO {output} '
-                        'SELECT {code}uid as geom_id, '
+                        'SELECT {name} as geom_name, '
+                        '       {code}uid as geom_id, '
                         '       wkb_geometry as geom '
                         'FROM {input} '.format(
+                            name=GEOGRAPHY_PROPERNAMES[self.resolution()],
```

Comment:
This line should be `name=GEOGRAPHY_PROPERNAMES[self.resolution],` (removing the parentheses)